src/plugin.js

```diff
@@ -139,13 +143,13 @@ export default (o, Dayjs, dayjs) => {
     }
     switch (unit) {
       case C.DATE:
-        instanceFactory(int, this.$jM)
+        innerSetDate(int, this.$jM)
         break
       case C.M:
-        instanceFactory(this.$jD, int)
+        innerSetDate(this.$jD, int === -1 ? 11 : int, int === -1 ? this.$jy - 1 : this.$jy)
```

Comment:
I think it would be useful if you describe this line use case in a comment, so anyone else can understand why `int === -1` is so special.

Reply:
Of course, will do.

Reply:
It was also causing another error related to changes to month which would change the year. Wrote another test for it as well.